### PR TITLE
Add frontend and static files fixes

### PR DESCRIPTION
**Pull request by tinashechiraya**
[Screencast from 27-10-2023 16:00:48.webm](https://github.com/kartoza/miniSASS/assets/70011086/2a9bd774-1b55-4689-b2be-fd9942170ec2)

### CODE REVIEW

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** FAILURE
**Build duration:** 49 sec

File: .example.env [hunk 1]
```diff
@@ -23,12 +23,6 @@ RESOLVER=127.0.0.11
 HTTPS_HOST=${COMPUTER_HOSTNAME}
 HTTPS_PORT=443
 
-# define media and static url
-MEDIA_ROOT=/django_project/minisass/static
```
Comment (NyakudyaA) [inline .example.env:27]: I think for the testing site we still need this to be set to `MEDIA_ROOT=/home/web/django_project/minisass/static/`

File: .example.env [hunk 2]
```diff
@@ -23,12 +23,6 @@ RESOLVER=127.0.0.11
...
-STATIC_ROOT=/django_project/minisass/media
```
Comment (NyakudyaA) [inline .example.env:28]: same as above

File: .example.env [hunk 3]
```diff
@@ -23,12 +23,6 @@ RESOLVER=127.0.0.11
...
-MEDIA_URL=/media/
```
Comment (NyakudyaA) [inline .example.env:29]: Let's keep this in the env even if they use defaults, will help in future if they still need to be exclusively defined.

**Review by NyakudyaA — CHANGES_REQUESTED** [review]
Looks good, except that we still need those values defined in the env in order for this to work in testing

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** FAILURE
**Build duration:** 41 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** FAILURE
**Build duration:** 45 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** FAILURE
**Build duration:** 40 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** FAILURE
**Build duration:** 41 sec

File: .example.env (hunk 1 above)
Comment (tinashechiraya) [reply .example.env:27]: @NyakudyaA alright I have made the modifications

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 4 min 28 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** FAILURE
**Build duration:** 41 sec

**Pull request merged**